### DISCUSSION TITLE
Allow to develop on any Chromium-based Browser

### DISCUSSION THREAD

**Discussion by jolo-dev** [extension-js/extension.js#56] [Ideas]
To my shame, I don't use Chrome but another Chromium-based browser (Arc or Brave). Is there a possibility to use them instead?
If not, how about making the `--browser` more generic and pointing to the path of the browser?

**Comment by cezaraugusto** [comment]
Hi @jolo-dev, good one! For now you can't, but I like the idea of allowing a custom browser path for the `--browser` flag. My current focus is Firefox/Safari, but let's have an issue from this discussion so it can be done in a near future. Thanks!

**Reply by cezaraugusto** [reply]
there we go https://github.com/cezaraugusto/extension.js/issues/59
